server/planning/feed_parsers/onclusive.py

```diff
@@ -0,0 +1,116 @@
+import datetime
+import logging
+import arrow
+from superdesk import get_resource_service
+from superdesk.metadata.item import (
+    ITEM_TYPE,
+    CONTENT_TYPE,
+    GUID_FIELD,
+    CONTENT_STATE,
+)
+from superdesk.errors import ParserError
+
+logger = logging.getLogger(__name__)
+
+
+class Onclusive:
+    """
+    Superdesk event parser
+
+    Feed Parser which can parse the Onclusive API Events
+    """
+
+    NAME = "onclusiveapi"
+    label = "Onclusive API"
+    all_events = []
+
+    def parse_http(self, content, provider):
+        return self.parse(content, provider)
+
+    def parse(self, content, provider=None):
+        try:
+            for event in content:
+                guid = "urn:newsml:{}:{}".format(event["createdDate"], event["itemId"])
+
+                if get_resource_service("events").find_one(req=None, guid=guid):
```

Comment:
I think we don't need this one, imo it should or will be supported right @MarkLark86 ?

Reply:
@petrjasek This is similar to Events ingest, until we come up with a strategy to allow management of the data from both the source and within Superdesk we decided to not allow updates for now.

IMO we can go forth with not allowing for now (so we can get the ingest system tested), and later on re-address the issue of updating certain metadata when ingesting an item with the same ID

Reply:
ok